src/utils.ts

```diff
@@ -78,3 +78,18 @@ export function logPayload(type: string, payload: any): string {
       return `[${type}]`;
   }
 }
+
+export async function sendMessage(
```

Comment:
I would resolve values on test level and pass to this method instead passing cucumber context here